internal/cli/cmd/verify/stage.go

```diff
@@ -0,0 +1,134 @@
+package verify
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"connectrpc.com/connect"
+	"github.com/spf13/cobra"
+
+	"github.com/akuity/kargo/internal/cli/client"
+	"github.com/akuity/kargo/internal/cli/config"
+	"github.com/akuity/kargo/internal/cli/option"
+	v1alpha1 "github.com/akuity/kargo/pkg/api/service/v1alpha1"
+)
+
+type verifyStageOptions struct {
+	Config        config.CLIConfig
+	ClientOptions client.Options
+
+	Project string
+	Name    string
+	Abort   bool
+}
+
+func newVerifyStageCommand(cfg config.CLIConfig) *cobra.Command {
```

Comment:
This is another inconsistency we can fix when time permits. (Not specifically introduced here.)

These constructor-like funcs for building cobra commands are sometimes named like `newNounCommand` and other times are named like `newVerbNounCommand`.

I'll open a separate issue.